lib/install/management/delete.go

```diff
@@ -46,11 +47,36 @@ func (d *Dispatcher) DeleteVCH(conf *config.VirtualContainerHostConfigSpec) erro
 		return nil
 	}
 
-	if err = d.DeleteVCHInstances(vmm, conf); err != nil {
-		// if container delete failed, do not remove anything else
-		log.Infof("Specify --force to force delete")
+	d.parentResourcepool, err = d.getComputeResource(vmm, conf)
+	if err != nil {
+		log.Error(err)
+		if !d.force {
+			log.Infof("Specify --force to force delete")
+			return err
+		}
+		// Assume the cVMs and RP have already been deleted
+		log.Warnf("Proceeding with delete of VCH due to --force")
+		err = nil
+	}
+	defaultrp, err := d.session.Cluster.ResourcePool(d.ctx)
+	if err != nil {
 		return err
 	}
+	if d.parentResourcepool != nil {
+		if d.parentResourcepool.Reference() == defaultrp.Reference() {
+			return fmt.Errorf("Targed VCH is in cluster's default resource pool. Refusing to delete it.")
```

Comment:
Why would we not remove if it's in the cluster default?  We won't install there, so it's been moved there...why not remove it?

Reply:
I thought this was a requirement, but that makes sense to me. I removed this

Reply:
Well..hell...looking at the issue I see why you'd think that was a requirement..I think the first item should have said "vic-machine _create_ will fail if the target rp is the default rp"...was really thinking of adding a new validation rule.  I'm not sure that's really needed...

I think we prevent the install into a cluster default RP by creating an RP on each deploy.  Previously there was an issue that would (in some cases) result in a VCH deployed to the default RP, but I think since 1.2.1 that's fixed...

Reply:
yea you're right 👍